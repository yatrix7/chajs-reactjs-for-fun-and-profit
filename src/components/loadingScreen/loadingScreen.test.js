import React from 'react'
import renderer from 'react-test-renderer'
import LoadingScreen from './loadingScreen'

it('renders correctly with weight', () => {
	const tree = renderer.create(<LoadingScreen />).toJSON()
	expect(tree).toMatchSnapshot()
})
