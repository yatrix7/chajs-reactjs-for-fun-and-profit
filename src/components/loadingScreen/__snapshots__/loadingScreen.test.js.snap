// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with weight 1`] = `
<div
  className="loading-screen"
>
  <svg
    className="svg"
    viewBox="0 0 100 100"
    width="400px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="50"
      cy="50"
      fill="none"
      r="45"
      stroke="#0000FF"
      strokeWidth="4"
    />
    <circle
      cx="50"
      cy="50"
      fill="none"
      r="33"
      stroke="#0000cc"
      strokeWidth="20"
    />
    <circle
      cx="50"
      cy="50"
      fill="none"
      r="21"
      stroke="#0000FF"
      strokeWidth="4"
    />
    <circle
      cx="50"
      cy="50"
      fill="none"
      r="14"
      stroke="#323232"
      strokeWidth="12"
    />
    <text
      className="fo-five"
    >
      <tspan
        x="13"
        y="51"
      >
        45
      </tspan>
      <tspan
        x="12"
        y="57"
      >
        lbs
      </tspan>
    </text>
    <text
      className="upside-down-fo-five"
    >
      <tspan
        x="13"
        y="50"
      >
        45
      </tspan>
      <tspan
        x="12"
        y="57"
      >
        lbs
      </tspan>
    </text>
  </svg>
</div>
`;
